homeassistant/components/notify/xmpp.py

```diff
@@ -10,7 +10,10 @@
     ATTR_TITLE, DOMAIN, BaseNotificationService)
 from homeassistant.helpers import validate_config
 
-REQUIREMENTS = ['sleekxmpp==1.3.1', 'dnspython3==1.12.0']
+REQUIREMENTS = ['sleekxmpp==1.3.1',
+                'dnspython3==1.12.0',
+                'pyasn1==0.1.9',
+                'pyasn1-modules==0.0.8']
```

Comment:
No, because because otherwise the requirements don't get installed. As far as I remember was this the reason to add `dnspython3` as requirement in the first place. A quick test shows:

```
[fab@mob03485 ~]$ pyvenv-3.4 test-xmpp && cd test-xmpp
[fab@mob03485 test-xmpp]$ source ./bin/activate
(test-xmpp) [fab@laptop019 test-xmpp]$ pip3 install sleekxmpp
Collecting sleekxmpp
Installing collected packages: sleekxmpp
Successfully installed sleekxmpp-1.3.1
(test-xmpp) [fab@mob03485 test-xmpp]$ pip3 freeze
sleekxmpp==1.3.1
```